client/lib/cordcharts.ts

```diff
@@ -243,9 +243,112 @@ const chordcharts: ChordCharts = {
     title: "G minor"
   }
 
+    // Minor accidentals Chords
+  "A#m": {
+    "fingers": [
+      [1, 0],
+      [2, 2],
+      [3, 3],
+      [4, 3],
+      [5, 0],
+      [6, 0]
+    ],
+    "barres": [
+      "fromString" : 6,
+      "toString": 1, 
+      "fret": 0
+    ],
+    title: "A# minor (Bb minor)"
+  },
+
+// for B#m, just show Cm chord
+  
+  "B#m": {
+    "fingers": [
+      [1, 3],
+      [2, 4],
+      [3, 5],
+      [4, 5],
+      [5, 3]
+    ],
+    "barres": [
+    ],
+    title: "B# minor (C minor)"
+  },
+
+  "C#m": {
+    "fingers": [
+      [1, 0],
+      [2, 5],
+      [3, 6],
+      [4, 6],
+      [5, 0]
+    ],
+    "barres": [
+      "fromString" : 6,
+      "toString": 1, 
+      "fret": 4
+    ],
+    title: "C# minor (Db minor)"
+  },
+  "D#m": {
+    "fingers": [
+      [1, 2],
+      [2, 4],
+      [3, 3],
+      [4, 0]
+    ],
+    "barres": [
+    ],
+    title: "D# minor (Eb minor)"
+  },
+
+  // return Fm for E#m 
+  "E#m": {
+    "fingers": [
+      [4, 3],
+      [5, 3]
+    ],
+    "barres": [
+      {
+        "fromString": 6,
+        "toString": 1,
+        "fret": 1
+      }
+    ],
+    title: "E# minor (F minor)"
+  },
+
+  "F#m": {
+    "fingers": [
+      [2, 2], 
+      [3, 2],
+      [4, 4]
+    ],
+    "barres": [
```

Comment:
Need to add 2nd fret barre

Reply:
was using this variation
https://imgur.com/a/nd6GkNE

Reply:
changed to this shape https://imgur.com/a/qg5lVNC